containers/Connector/Connector.tsx

```diff
@@ -70,12 +72,14 @@ const useConnector = () => {
 	}, [signer, dispatch]);
 
 	useEffect(() => {
-		const rateInterval = setInterval(() => {
-			dispatch(setExchangeRates());
-		}, 15000);
+		const ratesListener = (exchangeRates: Rates) => {
+			dispatch(setExchangeRates(exchangeRates));
+		};
+
+		sdk.events.on('exchangeRates_updated', ratesListener);
```

Comment:
Would be nice if we improved the api here to something like `sdk.exchange.onRatesUpdated()` rather than having to call on directly and pass the keys.